### ISSUE TITLE
No easy way to generate stubs for django apps

### ISSUE THREAD

**Issue by sebastian-philipp** [Instagram/MonkeyType#137]
I understand that you've already documented steps to generate stubs for django applications. And that's awesome!

Unfortunately the steps are too complicated and I need to give up using type hints for now. :cry:

**Comment by carljm** [comment]
Yes, our documentation is mostly reference-style, and MonkeyType's configuration is very flexible to handle many different possible situations. We could do better with tutorial-style walk-through documentation for typical cases, including a Django app. Thanks for the report!

**Comment by sebastian-philipp** [comment]
My use case was: I wanted to validate a big [changeset](https://github.com/ASKBOT/askbot-devel/compare/master...ASKBOT:0.11.x) without going to much into the details of how I get there, especially as this was a very time constrained project
